rust/src/bindgen.rs

```diff
@@ -538,16 +538,16 @@ impl<'a> Instantiator<'a> {
     fn instantiate_static_module(&mut self, idx: StaticModuleIndex, args: &[CoreDef]) {
         let i = self.instances.push(idx);
         let core_file_name = self.gen.core_file_name(&self.name, idx.as_u32());
-        self.gen.init.pyimport("pathlib", None);
+        self.gen.init.pyimport("importlib_resources", None);
 
         uwriteln!(
             self.gen.init,
-            "path = pathlib.Path(__file__).parent / ('{}')",
+            "file = importlib_resources.files() / ('{}')",
             core_file_name,
         );
         uwriteln!(
             self.gen.init,
-            "module = wasmtime.Module.from_file(store.engine, path)"
+            "module = wasmtime.Module(store.engine, file.read_bytes())"
```

Comment:
Is it possible to test whether `file` exists on the filesystem? Being able to plumb the file-ness all the way down can sometimes be useful in terms of various optimizations. In this case I don't think it changes anything today, but it might enable Wasmtime to do various tricks in the future.